README.md

```diff
@@ -214,6 +214,11 @@ const App = () => (
 export default App;
 ```
 
+#### Access inner component instance
+
+With theming version 3 we implemented the new `forwardRef` API.
```

Comment:
Since we don't have both versions at the same time, I would just document the status quo and the rest is in changelog.

Reply:
Done.